src/integration/java/io/pinecone/PineconeIndexOperationsClientIntegrationTest.java

```diff
@@ -48,5 +48,6 @@ public void createAndDelete() throws IOException {
 
         // Cleanup
         pinecone.deleteIndex(indexName);
+        Thread.sleep(3500);
```

Comment:
We must have a timeout after index deletion is called so the next test doesn't pick up this indexName, since deletion is not an atomic process.

Reply:
Discussed this internally on why its needed!